# Gripper.srv
bool data
---
float32 angle